packages/runtime-common/index.ts

```diff
@@ -227,7 +230,7 @@ export async function chooseCard<T extends BaseDef>(
   }
   let chooser: CardChooser = here._CARDSTACK_CARD_CHOOSER;
 
-  return await chooser.chooseCard<T>(query, opts);
+  return await chooser.chooseCard<T>(owner, query, opts);
```

Comment:
This needs an owner? This remains as resource bcos it bottoms out to `getSearch`. Is there any intention to not have this as a resource?

Reply:
remember we are dealing in card resource now. it's not safe to pass raw instances around. that means that if you want a card you need to say who wants it because it'g gonna give you a resource for the card, not the raw card.